particles/Recommend/Recommend.js

```diff
@@ -18,6 +18,7 @@
         populationView.toList().then(data => {
           views.get('recommendations').store(data[0]);
           views.get('recommendations').store(data[1]);
+          views.get('recommendations').store(data[2]);
```

Comment:
it was kind of nice that Recommend particle filtered out some of the elements added by WishlistFor and the resulting lists weren't identical, so maybe add one more element to WishlistFor?